lib/galaxy/objectstore/cloud.py

```diff
@@ -620,19 +687,20 @@ def get_store_usage_percent(self):
         return 0.0
 
 
-class SwiftObjectStore(S3ObjectStore):
+class SwiftObjectStore(CloudObjectStore):
     """
     Object store that stores objects as items in a Swift bucket. A local
     cache exists that is used as an intermediate location for files between
     Galaxy and Swift.
     """
 
     def _configure_connection(self):
+        # TODO: Replace with Cloudbridge connection.
         log.debug("Configuring Swift Connection")
```

Comment:
If it is already addressed in the super class - can you just delete the code and the comment?